internal/server/server.go

```diff
@@ -90,8 +92,16 @@ func Serve(
 	if err != nil {
 		return err
 	}
+
 	httpMux.Handle("/jwks.json", jwksHandler)
 
+	spaHandler, err := spa.Handler(ui.Assets, "dist/ui", "index.html", false)
+	if err != nil {
+		fmt.Println("Failed to load spa:", err)
```

Comment:
Can we use an existing logger struct with `warn` tag?

Reply:
added `logger.Warn`